Joedmin-hw3/Joedmin-hw3/Program.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+
+namespace Joedmin_hw3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+                Console.Write("What do you want to do? login/register/exit: ");
+                string answer = Console.ReadLine();
+
+                // Exit
+                if (String.Equals(answer, "exit", StringComparison.OrdinalIgnoreCase))
+                {
+                    Process.GetCurrentProcess().Kill();
```

Comment:
Use `Environment.Exit` for this kind of thing.  Also, since you're in `Main` you could just do `return;`

Reply:
Ohh, ok, I found this in the comments bellow [this](https://stackoverflow.com/a/4019622/14466874) answer